README.md

```diff
@@ -0,0 +1,3 @@
+# check_copyright
+
+Espressif repository for the script to check SPDX License Header in the files. 
```

Comment:
Hi @kumekay,
thank you for the suggestion.

The `create-project-tools` tool currently allows you to use `repository-templates` when creating a new Gitlab/GitHub project.

The `python-project` repository template also includes a `pre-commit-config.yml` file, [see Gitlab](https://gitlab.espressif.cn:6688/espressif/repository-templates/python-project/-/blob/master/.pre-commit-config.yaml).

Do you mean that this should be done by updating this repository template (adding the `check_copyright.py` script and pre-commit config in the same way as it is currently defined in the `esp-idf` project)?

Reply:
@tomassebestik  I think it may be helpful not only for python projects but for C projects too.

We are going to replace the script in IDF with this one.